app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    interim = command.split()
+    command_file = interim[0]
+    copied_file = interim[1]
+    file_to_copy = interim[2]
```

Comment:
the way you wrote is even worse than the previous solution
```suggestion
    command, copied_file, file_to_copy = command.split()
```
`If len(command.split())` not equal to 3, your code will broke
I advise you to add this check or `try ... exсept`